workflows/envs/cytotable.yaml

```diff
@@ -7,4 +7,4 @@ dependencies:
   - python==3.8
```

Comment:
A bit outside the scope of this PR, but wanted to mention just in case: it may be possible to increase the Python version here if CytoTable is the only dependency (versions < 3.11).

Reply:
updated to 3.10!